tests/suites/test_suite_ecp.function

```diff
@@ -698,8 +711,7 @@ void ecp_tls_write_read_point( int id )
     memset( buf, 0x00, sizeof( buf ) ); vbuf = buf;
     TEST_ASSERT( mbedtls_ecp_tls_write_point( &grp, &grp.G,
                     MBEDTLS_ECP_PF_COMPRESSED, &olen, buf, 256 ) == 0 );
-    TEST_ASSERT( mbedtls_ecp_tls_read_point( &grp, &pt, &vbuf, olen )
-                 == MBEDTLS_ERR_ECP_FEATURE_UNAVAILABLE );
+    TEST_ASSERT( mbedtls_ecp_tls_read_point( &grp, &pt, &vbuf, olen ) == 0 );
```

Comment:
Now that this works, can we also check that we're getting the expecting values here (like we're doing in the next block for uncompressed)?

Reply:
:rocket: